src/colmap/controllers/incremental_mapper.cc

```diff
@@ -237,12 +236,12 @@ BundleAdjustmentOptions IncrementalMapperOptions::GlobalBundleAdjustment()
   options.solver_options.max_linear_solver_iterations = 100;
   options.solver_options.logging_type =
       ceres::LoggingType::PER_MINIMIZER_ITERATION;
-  options.solver_options.minimizer_progress_to_stdout = true;
+  options.solver_options.minimizer_progress_to_stdout = false;
```

Comment:
Please, consider not disabling minimizer progress printing if verbose output is requested.

Reply:
Yes, this makes sense. I would agree.

Reply:
There is VLOG_IS_ON(1) that can be used here.

Reply:
This is already the case: if false, Ceres logs to VLOG(1) instead of stdout

Reply:
We could log to stdout instead with VLOG_IS_ON, but then it doesn’t get written to additional sinks (eg file)

Reply:
Got it. Thanks.